client/hijack.go

```diff
@@ -65,8 +65,8 @@ func (cli *Client) setupHijackConn(req *http.Request, proto string) (_ net.Conn,
 		}
 	}
 
-	ctx, span := tp.Tracer("").Start(ctx, req.Method+" "+req.URL.Path)
-	span.SetAttributes(semconv.HTTPClientAttributesFromHTTPRequest(req)...)
+	ctx, span := tp.Tracer("").Start(ctx, req.Method+" "+req.URL.Path, trace.WithSpanKind(trace.SpanKindClient))
```

Comment:
@cpuguy83 I added `trace.WithSpanKind(trace.SpanKindClient)` here


Do you think we should look at using containerd's tracing package as well (in a follow-up)? I can make create a tracking issue for that if it makes sense to do.